packages/editor/src/components/post-publish-panel/style.scss

```diff
@@ -54,10 +54,9 @@
 }
 
 .editor-post-publish-panel__link {
-	color: $blue-medium-700;
 	font-weight: 400;
 	padding-left: 4px;
-	text-decoration: underline;
+	text-decoration: none;
```

Comment:
Minor: I guess given that the element is a span, we don't need to add text-decoration: none; and we can just remove text-decoration: underline;.

Reply:
Agreed, this PR works fine with the text-decoration rule deleted. 👍